apps/web/src/app/[lang]/app/[type]/management/data.tsx

```diff
@@ -339,6 +347,95 @@ export const dataConfig: DataConfigArray = {
             "concurrencyStamp",
           ],
           schema: $Volo_Saas_Host_Dtos_EditionDto,
+          actionList: () => [
+            {
+              type: "Dialog",
+              cta: "Move All Tenants",
+              description: "Move All Tenants",
+              componentType: "Autoform",
+              autoFormArgs: {
+                submit: {
+                  cta: "Move All Tenants",
+                },
+                formSchema: z.object({
+                  editionId: z.string(),
+                }),
+                fieldConfig: {
+                  editionId: {
+                    renderer: function EditionComboboxRenderer(
+                      props: AutoFormInputComponentProps,
+                    ) {
+                      const [editionList, setEditionList] = useState<
+                        Volo_Saas_Host_Dtos_EditionDto[]
+                      >([]);
+                      const [errorMessage, setErrorMessage] =
+                        useState<ErrorTypes | null>(null);
+
+                      useEffect(() => {
+                        const fetchEditions = async () => {
+                          const editions = await getAllEditionsApi();
+                          if (editions.type === "success") {
+                            const updatedEditionList: Volo_Saas_Host_Dtos_EditionDto[] =
+                              [
+                                { id: "", displayName: "unassigned edition" },
+                                ...editions.data,
+                                // TODO: Add a filter to make the original edition disappear
+                              ];
+                            setEditionList(updatedEditionList);
+                          } else {
+                            setErrorMessage(editions);
+                          }
+                        };
+                        void fetchEditions();
+                      }, []);
+
+                      return (
+                        <div>
+                          {!errorMessage ? (
+                            <CustomCombobox<Volo_Saas_Host_Dtos_EditionDto>
+                              childrenProps={props}
+                              emptyValue="Select Edition"
+                              list={editionList}
+                              selectIdentifier="id"
+                              selectLabel="displayName"
+                            />
+                          ) : (
+                            <div className="text-muted-foreground text-md text-center">
+                              {errorMessage.message === "error" ||
+                              errorMessage.message === "api-error"
+                                ? errorMessage.message
+                                : " An error occurred please try again later."}
```

Comment:
if it falls into this block then we are sure it's an error and we have a the message as error or API error, it would be better to have such a code: 

```suggestion
                              { `${errorMessage.type}` + (errorMessage.message ||  " An error occurred please try again later.")}
```

Reply:
This code show the error type and the message provided by it, if there is no message, we show the default one.